server/game/cards/13-O4B/Fort51.js

```diff
@@ -0,0 +1,66 @@
+const GameActions = require('../../GameActions/index.js');
+const OutfitCard = require('../../outfitcard.js');
+
+class Fort51 extends OutfitCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onGadgetInvented: event => event.gadget.hasKeyword('Weapon') || event.gadget.getType() === 'dude' && event.player === this.controller
+            },
+            handler: context => {
+                this.game.promptForSelect(context.player, {
+                    activePromptTitle: 'Select a dude',
+                    waitingPromptTitle: 'Waiting for opponent to select dude',
+                    cardCondition: card => card.location === 'play area' && card.controller !== this.controller && card.influence < context.event.gadget.cost,
+                    cardType: 'dude',
+                    gameAction: 'addBounty',
+                    onSelect: (player, card) => {
+                        this.game.resolveGameAction(GameActions.addBounty({ card, amount: 1 }), context);
+                        this.game.addMessage('{0} gives {1} 1 bounty thanks to {2}', player, card, this);
```

Comment:
Put the `addMessage` in `thenExecute` after the `resolveGameAction`.

Reply:
Done.